README.md

```diff
@@ -62,13 +62,14 @@ npm install -g esy@0.5.7
 - `esy install`
 - `esy '@test' install`
 
-### Building
+### Building from Head
```

Comment:
What does "from Head" mean here? Isn't that usually where you run tests, and shouldn't it also work anyways?

Reply:
Good call. I think I was trying to call out how esy build works wrt to esy build vs. esy '@test' build. I'm still wrapping my head around it, but my understanding is that esy build vs. est '@test' build make different environments/packages/etc.

This could be wrong, I've been using esy for 3 days now, so open to suggestions/advice!

Reply:
Oh, right. Yeah I think "sandboxes" might be more accurate than environments, which as defined by esy seems to be something else.

Reply:
So `esy install` will install the default sandbox, while `esy @test install` will install the test sandbox.